### PR TITLE
Update cribsheet docs

### PR DESCRIPTION
**Pull request by macdonst**
Signed-off-by: macdonst <simon.macdonald@gmail.com>

### CODE REVIEW

File: app/docs/md/enhance-styles/cribsheet.md [hunk 1]
```diff
@@ -2,7 +2,35 @@
...
 Just start your project (`npm start`) and navigate to `/_styleguide/cribsheet`. Here you'll be able to search through all the utility classes and custom properties available to your app. Any [customizations](/docs/enhance-styles/customization) you make to your styleguide will automatically be reflected in your cribsheet.
 
+If you are [migrating from an Architect project](https://enhance.dev/cookbook/migrate-from-architect) to an Enhance project or you are starting with our [minimal template](https://github.com/enhance-dev/enhance-starter-project-minimal) then you will need to add the cribsheet to your project.
+
+## Install Dependencies
+
+First install the `@enhance/styles-cribsheet` package
+
+```bash
+npm install @enhance/styles-cribsheet
```
Comment (colepeters) [inline app/docs/md/enhance-styles/cribsheet.md:16]: I think it's also required that users install Enhance Styles as well, as it won't be included in the minimal template or an Arc migration (necessarily)?

File: public/css/docs-highlight.css [hunk 2]
```diff
@@ -52,8 +52,8 @@
   --hl-namespace: var(--dk-pink);
 
   --hl-highlight-line: #202641;
-  --hl-deletion: #461e1f;
-  --hl-addition: #14301d;
+  --hl-deletion: #ec5f67;
+  --hl-addition: #99c794;
```
Comment (colepeters) [inline public/css/docs-highlight.css:56]: Where'd these come from? 🧑🏻‍🎨

File: app/docs/md/enhance-styles/cribsheet.md (hunk 1 above)
Comment (macdonst) [reply app/docs/md/enhance-styles/cribsheet.md:16]: 🤦  of course. Which leads me to the question, do we even need this update?

File: public/css/docs-highlight.css (hunk 2 above)
Comment (macdonst) [reply public/css/docs-highlight.css:56]: From begin.com hljs diff colors. If you look at a diff in the docs you will not be able to read it.

File: app/docs/md/enhance-styles/cribsheet.md (hunk 1 above)
Comment (colepeters) [reply app/docs/md/enhance-styles/cribsheet.md:16]: Hmmm… I guess maybe not? 😂

**Pull request closed without merging**